### PR TITLE
Update `amounts-expenses` page for multi-member flow

### PR DESCRIPTION
**Pull request by jessieay**
* [Finishes #152510521]
* We did not add any validations to these inputs, just changed the
wording and design and in some cases moved the field to the member class

screen shot 2017-11-09 at 4 05 00 pm

screen shot 2017-11-09 at 4 04 32 pm

### CODE REVIEW

File: app/controllers/medicaid/amounts_expenses_controller.rb [hunk 1]
```diff
@@ -1,11 +1,47 @@
 module Medicaid
-  class AmountsExpensesController < MedicaidStepsController
+  class AmountsExpensesController < Medicaid::MemberStepsController
+    def current_member
+      @_current_member ||= super || first_member_with_expenses
+    end
+
+    def update
```
Comment (hartsick) [inline app/controllers/medicaid/amounts_expenses_controller.rb]: one thing @disambiguator and I had been starting to try was using the `after_successful_update_hook` and defining model-specific params from the step_params (e.g. `member_params` that could just be `step_params.merge(member_id: current_member.id)`). the idea behind that is that it exercises our existing interface, surfaces pain points, and allows us to have a contained site for change in future. may be worth trying here?

File: app/controllers/medicaid/expenses_alimony_controller.rb [hunk 2]
```diff
@@ -1,4 +1,18 @@
 module Medicaid
   class ExpensesAlimonyController < MedicaidStepsController
+    private
+
+    def after_successful_update_hook
```
Comment (hartsick) [inline app/controllers/medicaid/expenses_alimony_controller.rb]: woo

File: db/migrate/20171109193950_add_expense_amounts_to_members.rb [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+           child_support_alimony_arrears_expenses=primary_medicaid_members.child_support_alimony_arrears_expenses,
+           self_employment_expenses=primary_medicaid_members.self_employed_monthly_expenses
+        FROM (
+          SELECT min(id) as id, benefit_application_id, unemployment_income
+          FROM members
+          WHERE benefit_application_type = 'MedicaidApplication'
+          GROUP BY benefit_application_id, unemployment_income
+        )
+        AS primary_medicaid_members
+        WHERE medicaid_applications.id = primary_medicaid_members.benefit_application_id
```
Comment (hartsick) [inline db/migrate/20171109193950_add_expense_amounts_to_members.rb]: these are so wild 🎈

**Comment by jessieay** [comment]
specs should be green now. Feel free to merge tomorrow if it gets approved @disambiguator !

File: app/controllers/medicaid/amounts_expenses_controller.rb (hunk 1 above)
Comment (jayroh) [reply app/controllers/medicaid/amounts_expenses_controller.rb]: I like the idea of `member_params`. One slight suggestion I have is to give it one nudge towards more specificity and call it `member_step_params`. That'd keep the `step` context in line.

Comment (disambiguator) [reply app/controllers/medicaid/amounts_expenses_controller.rb]: @jayroh Dig it.

We should move in that direction, although on second look this one deviates just ever so slightly from that pattern and we can't just do the quick refactor right now. Gonna save this one for later.

**Pull request merged**